plugins/hydra_ray_launcher/tests/test_ray_launcher.py

```diff
@@ -36,40 +32,40 @@ def test_discovery() -> None:
     ]
 
 
-# @pytest.mark.skipif(sys.platform.startswith("win"), reason=win_msg)
-# @pytest.mark.parametrize("launcher_name, overrides", [("ray", [])])
-# class TestRayLauncher(LauncherTestSuite):
-#     """
-#     Run the Launcher test suite on this launcher.
-#     """
-#
-#     pass
+@pytest.mark.skipif(sys.platform.startswith("win"), reason=win_msg)
+@pytest.mark.parametrize("launcher_name, overrides", [("ray", [])])
+class TestRayLauncher(LauncherTestSuite):
+    """
+    Run the Launcher test suite on this launcher.
+    """
+
+    pass
 
 
-# @pytest.mark.skipif(sys.platform.startswith("win"), reason=win_msg)
-# @pytest.mark.parametrize(
-#     "task_launcher_cfg, extra_flags",
-#     [
-#         (
-#             {
-#                 "defaults": [
-#                     {"hydra/launcher": "ray"},
-#                     {"hydra/hydra_logging": "hydra_debug"},
-#                     {"hydra/job_logging": "disabled"},
-#                 ],
-#                 "hydra": {
-#                     "launcher": {
-#                         "cls": "hydra_plugins.hydra_ray_launcher.ray_launcher.RayLauncher",
-#                     }
-#                 },
-#             },
-#             ["-m"],
-#         )
-#     ],
-# )
-# class TestRayLauncherIntegration(IntegrationTestSuite):
-#     """
-#     Run this launcher through the integration test suite.
-#     """
-#
-#     pass
+@pytest.mark.skipif(sys.platform.startswith("win"), reason=win_msg)
+@pytest.mark.parametrize(
+    "task_launcher_cfg, extra_flags",
+    [
+        (
+            {
+                "defaults": [
+                    {"hydra/launcher": "ray"},
+                    {"hydra/hydra_logging": "hydra_debug"},
+                    {"hydra/job_logging": "disabled"},
+                ],
+                "hydra": {
+                    "launcher": {
+                        "cls": "hydra_plugins.hydra_ray_launcher.ray_launcher.RayLauncher",
```

Comment:
This is probably not needed, `hydra/launcher: ray` should do it.
Also, is this testing the local mode or the aws mode?